sessionctx/variable/session.go

```diff
@@ -646,6 +646,9 @@ type SessionVars struct {
 
 	// EnableSlowLogMasking indicates that whether masking the query data when log slow query.
 	EnableSlowLogMasking bool
+
+	// EnableParallelApply indicates that thether to use parallel apply.
+	EnableParallelApply bool
```

Comment:
Should we add some variables to control the number of concurrency of the `apply`?

Reply:
No, after this https://github.com/pingcap/tidb/pull/16999, we use a single `ExecutorConcurreny` to control the concurrency of all types of executors.